src/Bridge/Doctrine/MongoDbOdm/Filter/SearchFilter.php

```diff
@@ -149,10 +141,25 @@ protected function filterProperty(string $property, $value, Builder $aggregation
             return;
         }
 
+        $this->addEqualityMatchStrategy($strategy, $aggregationBuilder, $matchField, $doctrineTypeField, $values, $caseSensitive);
+    }
+
+    /**
+     * Adds where clause according to the strategy.
+     *
+     * @throws InvalidArgumentException If strategy does not exist
+     */
+    protected function addEqualityMatchStrategy(string $strategy, Builder $aggregationBuilder, string $matchField, $fieldType, $values, bool $caseSensitive)
```

Comment:
Since the class is final, it should be private.

Reply:
So we allow to extends the ORM version but not the ODM one ? :/

Reply:
Yes. Because the ORM version will also be a final class in the next major version (3.0).